pkg/antctl/raw/multicluster/init.go

```diff
@@ -0,0 +1,100 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+type initOptions struct {
+	namespace  string
+	clusterSet string
+	clusterID  string
+}
+
+var initOpts *initOptions
+
+func (o *initOptions) validate() error {
+	if o.namespace == "" {
+		return fmt.Errorf("Namespace is required")
+	}
+	if o.clusterSet == "" {
+		return fmt.Errorf("ClusterSet is required")
+	}
+	if o.clusterID == "" {
+		return fmt.Errorf("clusterID is required")
+	}
+	return nil
+}
+
+var initExample = strings.Trim(`
+# Initialize the leader cluster with the given Namespace and ClusterSet. One ClusterSet and two ClusterClaims will be 
+created in the given Namespace.
+  $ antctl mc init --namespace=<NAMESPACE> --clusterset=<CLUSTERSET_ID> --clusterID=<CLUSTERID>
+`, "\n")
+
+func NewInitCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "init",
+		Short:   "Initialize the leader cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Example: initExample,
+		RunE:    initRunE,
+	}
+
+	o := initOptions{}
+	initOpts = &o
+	command.Flags().StringVar(&o.namespace, "namespace", "", "Namespace of the Antrea multi-cluster")
+	command.Flags().StringVar(&o.clusterSet, "clusterset", "", "ClusterSet of the leader cluster")
+	command.Flags().StringVar(&o.clusterID, "clusterID", "", "ClusterID of the leader cluster")
+
+	return command
+}
+
+func initRunE(cmd *cobra.Command, args []string) error {
+	if err := initOpts.validate(); err != nil {
+		return err
+	}
+	k8sClient, err := common.NewClient(cmd)
+	if err != nil {
+		return err
+	}
+	createdRes := []*unstructured.Unstructured{}
+	var createErr error
+	defer func() {
+		if createErr != nil {
+			if err := common.Rollback(cmd, k8sClient, createdRes); err != nil {
+				fmt.Fprintf(cmd.OutOrStdout(), "Failed to rollback: %v\n", err)
+			}
+		}
+	}()
+	createErr = common.CreateClusterClaim(cmd, k8sClient, initOpts.namespace, initOpts.clusterSet, initOpts.clusterID, &createdRes)
+	if createErr != nil {
+		return createErr
+	}
+	createErr = common.CreateClusterSet(cmd, k8sClient, initOpts.namespace, initOpts.clusterSet, "", "", "", initOpts.clusterID, initOpts.namespace, &createdRes)
+	if createErr != nil {
+		return createErr
+	}
+	fmt.Fprintf(cmd.OutOrStdout(), "Successfully initialized the leader cluster\n")
```

Comment:
I think we should print out the default token for members to join, just like "kubeadm init".

Reply:
The Secret looks like this: 
```bash
-----BEGIN CERTIFICATE-----
MIIC/jCCAeagAwIBAgIBADANBgkqhkiG9w0BAQsFADAVMRMwEQYDVQQDEwprdWJl
cm5ldGVzMB4XDTIyMDcyMDA2MDcxNVoXDTMyMDcxNzA2MDcxNVowFTETMBEGA1UE
AxMKa3ViZXJuZXRlczCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAOqp
dDZw6sugfGPNN2sE29xhuuwNko3d2h/Lf3jgHp0GfxEELW1BuCuKp7LjHuyzrLRq
3jkp0MYHn2XnI1L3jgyTdWyZaaXF+jLrK+J0Xkf672U2DnW8vqDdGrM8ogEkucj5
+eXDWe8u0zCnQQDUun+mcffAZJPnL/pzN7xCiMgX1mY0KZC8x15Or1ApyhFSGsGI
JNfzt1sADOefyNg6Qu+NAb56FwazuCFj8h9JgvLyJQjFlo6sE9yuz4QmTU3/XWWQ
aCidWbBo4tIN/OdA2ZSNEKjIYRE5hWzvG7JZ/m5C1oDfi88ehqOOiAs8AY10+gcP
BksILABg7NFLQ53Fb4kCAwEAAaNZMFcwDgYDVR0PAQH/BAQDAgKkMA8GA1UdEwEB
/wQFMAMBAf8wHQYDVR0OBBYEFIj9NG/ZslMcqVigQDPaoGAO+x89MBUGA1UdEQQO
MAyCCmt1YmVybmV0ZXMwDQYJKoZIhvcNAQELBQADggEBAJV/oXutYYV0q2X0koYh
YXxy0pf8ONYLPEjD549WViSeYA81IZ6xR2ubZFdmIRZdFgZi/r/WjXXIb7nADS0w
l4Y5/lyGqET7qb+jU7lzWn49NbwU7KRMfeS0+s4tylS8zsp6nXVdB6xppRWf4C8M
xsnzWoOmdi6P3PSWxIJyn5a2Szy25CXp4Ro2Pk7JIq+X0kD0kQvr08bwNf9g2Lsv
fkc+QWMba8UpVWl5XxoruIXX4pq0ahe/BkBQ59CKNKfAZv4hFmoScKFXraAfQwaf
9USmmOmBLoIuCYHsjhlZkDp8zlYmSkGdNl0X/ziJudffWHRtv6yzkvtsO6wcOQBJ
DVo=
-----END CERTIFICATE-----

eyJhbGciOiJSUzI1NiIsImtpZCI6IkVLZGI0RTFRQXpGRDFTTXlSYjA4WUJaQ2M4aDRWeTVfakdVQU0yR1dwXzAifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6InRlc3Qtc2VjcmV0LTIiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoidGVzdC1zYS0yIiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZXJ2aWNlLWFjY291bnQudWlkIjoiYmI2NDdhNmQtOTUwMy00MzY4LTgyZGQtZGQ0NjdiNTBjNzQzIiwic3ViIjoic3lzdGVtOnNlcnZpY2VhY2NvdW50OmRlZmF1bHQ6dGVzdC1zYS0yIn0.Ispw9AlRGsVEC7WZgxw_C0UxDw0j8D-KtVTC98J3msUIE7qhXGo5GGYzv_yD4Y4TSalPF4U8_lWblFFJ8YH52G-jvivblVlwVcl3PesBWX4vFHF70EgQVENL1xIf0lJfy723RFIw9-GtqcgVX0iiACZVCEJivg7JV6BSFCL8-yi18k5fG6DuytySSscSZTpxz5J2q8OEIHlxoWTJerZSXb12U_vhQmcjKk-mJnKSLE9-QAxKBjG2VxTW-ec9AfMUA-XVYW8jAnc2p2kcp7GQ5LRheb2JTrq7yqBbDYjaw8LtoV_O_fqWibpTWyf4KQh0eYo8CW4PW4r0CBh4-umOxg 
```

I'm worried it's too long for a command line argument to use.